src/org/ticket/controller/TicketController.java

```diff
@@ -0,0 +1,36 @@
+package org.ticket.controller;
+
+import org.ticket.model.Ticket;
+import org.ticket.model.dao.GenericDAO;
+
+import java.util.List;
+
+public class TicketController {
+    GenericDAO<Ticket> ticketDAO;
```

Comment:
As this, as it current is written, is available to other classes of the same package it would be a good idea to encapsulate this variable a private. Just a nit picking, not a serious problem.